packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -678,7 +704,7 @@ export const tasks = ({
       },
       {
         title: `Generating types ...`,
-        task: generateTypes,
+        task: () => generateTypes(),
```

Comment:
Why did you make this change?

Reply:
Types actually didn't generate. @dac09 has a more complete PR for this in https://github.com/redwoodjs/redwood/pull/4376 so I just added it so I could see my types generating.

Reply:
I know, but that's merged, and it looks exactly as the code you changed (`task: generateTypes,`) so it's worrying if that doesn't work.

Reply:
It does generate types now, I must not have had latest when I branched or something.

Reply:
I'd prefer to undo this change, to keep this file inline with the other generators

Reply:
Oh weird.

https://github.com/redwoodjs/redwood/pull/4376/files

I thought that PR used that syntax.

Ok. hmmm. I'll revert and check that types are still set.

Reply:
Ok, my mistake. Fixes.